src/paradigma/heart_rate/heart_rate_analysis.py

```diff
@@ -64,25 +65,84 @@ def signal_quality_classification(df: pd.DataFrame, config: SignalQualityClassif
 
     # Prepare the data
     lr_clf.feature_names_in_ = ['var', 'mean', 'median', 'kurtosis', 'skewness', 'f_dom', 'rel_power', 'spectral_entropy', 'signal_to_noise', 'auto_corr']
```

Comment:
I had a discussion before about normalizing the dominant frequency, and we decided to not scale it because the frequencies can simply be different for different people. Not sure how this works for you.

Reply:
I trained my classifier on a normalized dominant frequency for now, but I see the point. Lets discuss this another time ;)